.github/workflows/build.yml

```diff
@@ -148,24 +160,39 @@ jobs:
           
       - name: Run unit tests
         run: npm run test:unit
+        env:
+          CI: true
         working-directory: ./src/vscode-bicep
-        
+
       - name: Run E2E tests
+        run: npm run test:e2e
+        env:
+          CI: true
+        if: runner.os != 'Linux'
+        working-directory: ./src/vscode-bicep
+        
+      - name: Run E2E tests (Linux)
```

Comment:
>Linux [](start = 29, length = 5)

Can you add a comment that explains why we need a separate Linux path here?

Reply:
Yes, added a comment.